components/checkbox/Checkbox.razor

```diff
@@ -0,0 +1,19 @@
+﻿@namespace AntBlazor
+@inherits AntDomComponentBase
+
+<label class="ant-checkbox-wrapper" style="@Style;" id="@Id" @ref="Ref">
+    <span class="@ClassMapper.Class" style="">
+        <input @ref="_inputElement"
+               value="@(Value)"
+               checked="@(Checked)"
+               disabled="@(Disabled)"
+               @onchange="@InputCheckedChange"
+               @attributes="@InputAttributes"
+               type="checkbox"
+               class="ant-checkbox-input" />
+        <span class="ant-checkbox-inner"></span>
+    </span>
+    <span @ref="_contentElement">
```

Comment:
missing null-check

Reply:
null-check ??  what variable need to do null-check?